### PR TITLE
Microservice images are now licensed under GPL license

### PR DESCRIPTION
**Pull request by MattCzerner**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Docker images can not be distributed under MIT license while containing software licensed under GPL license. Because of that we license images under GPL license while source code stays under MIT license
|New feature| No
|BC breaks| No
|Fixes issues| 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by MattCzerner** [comment]
Full description on docker cloud is generated from README.md in repository. So we could not just add information there. Thats why LICENSE was renamed to LICENSE-source-coud and LICENSE-docker-image was created.

I added information about GPL license into short description on Docker cloud.
Here https://cloud.docker.com/u/shopsys/repository/docker/shopsys/microservice-product-search and here https://cloud.docker.com/u/shopsys/repository/docker/shopsys/microservice-product-search-export

File: microservices/product-search-export/README.md [hunk 1]
```diff
@@ -228,3 +228,10 @@ Please, check our [Contribution Guide](https://github.com/shopsys/shopsys/blob/m
 What to do when you are in troubles or need some help? The best way is to contact us on our Slack [http://slack.shopsys-framework.com/](http://slack.shopsys-framework.com/)
 
 If you want to [report issues](https://github.com/shopsys/shopsys/issues/new), please use the main [Shopsys repository](https://github.com/shopsys/shopsys).
+
+## License
+### Source Code
+Source code is under MIT license, for more information, read [LICENSE-source-code](https://shopsys/shopsys/blob/master/microservices/product-search-export/LICENSE-source-code).
+
+### Docker Image
+Built image is under GPL license, for more information read [LICENSE-docker-image](https://shopsys/shopsys/blob/master/microservices/product-search-export/LICENSE-docker-image)
```
Comment (boris-brtan) [inline microservices/product-search-export/README.md]:
```suggestion
Built image is under GPL license, for more information read [LICENSE-docker-image](https://github.com/shopsys/shopsys/blob/master/microservices/product-search-export/LICENSE-docker-image)
```

File: microservices/product-search-export/README.md [hunk 2]
```diff
@@ -228,3 +228,10 @@ Please, check our [Contribution Guide](https://github.com/shopsys/shopsys/blob/m
 What to do when you are in troubles or need some help? The best way is to contact us on our Slack [http://slack.shopsys-framework.com/](http://slack.shopsys-framework.com/)
 
 If you want to [report issues](https://github.com/shopsys/shopsys/issues/new), please use the main [Shopsys repository](https://github.com/shopsys/shopsys).
+
+## License
+### Source Code
+Source code is under MIT license, for more information, read [LICENSE-source-code](https://shopsys/shopsys/blob/master/microservices/product-search-export/LICENSE-source-code).
```
Comment (boris-brtan) [inline microservices/product-search-export/README.md]:
```suggestion
Source code is under MIT license, for more information, read [LICENSE-source-code](https://github.com/shopsys/shopsys/blob/master/microservices/product-search-export/LICENSE-source-code).
```

File: microservices/product-search/README.md [hunk 3]
```diff
@@ -97,3 +97,10 @@ Please, check our [Contribution Guide](https://github.com/shopsys/shopsys/blob/m
 What to do when you are in troubles or need some help? The best way is to contact us on our Slack [http://slack.shopsys-framework.com/](http://slack.shopsys-framework.com/)
 
 If you want to [report issues](https://github.com/shopsys/shopsys/issues/new), please use the main [Shopsys repository](https://github.com/shopsys/shopsys).
+
+## License
+### Source Code
+Source code is under MIT license, for more information, read [LICENSE-source-code](https://shopsys/shopsys/blob/master/microservices/product-search/LICENSE-source-code).
```
Comment (boris-brtan) [inline microservices/product-search/README.md]:
```suggestion
Source code is under MIT license, for more information, read [LICENSE-source-code](https://github.com/shopsys/shopsys/blob/master/microservices/product-search/LICENSE-source-code).
```

File: microservices/product-search/README.md [hunk 4]
```diff
@@ -97,3 +97,10 @@ Please, check our [Contribution Guide](https://github.com/shopsys/shopsys/blob/m
...
+
+### Docker Image
+Built image is under GPL license, for more information read [LICENSE-docker-image](https://shopsys/shopsys/blob/master/microservices/product-search/LICENSE-docker-image)
```
Comment (boris-brtan) [inline microservices/product-search/README.md]:
```suggestion
Built image is under GPL license, for more information read [LICENSE-docker-image](https://github.com/shopsys/shopsys/blob/master/microservices/product-search/LICENSE-docker-image)
```

**Pull request merged**